content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/dotnet-framework.md

```diff
@@ -183,20 +183,20 @@ If needed, configure the tracing library to send application performance telemet
 
 ## Custom instrumentation
 
-Your setup for custom instrumentation depends on your automatic instrumentation and includes additional steps depending on the method:
+Custom instrumentation depends on automatic instrumentation and includes additional steps depending on the method:
 
 {{< tabs >}}
 
 {{% tab "Windows" %}}
 
 <div class="alert alert-warning">
-  <strong>Note:</strong> If you are using both automatic and custom instrumentation, you must keep the package versions (for example: MSI and NuGet) in sync.
+  <strong>Note:</strong> Starting v3.0.0, custom instrumentation requires you also use automatic instrumentation. You should aim to keep both automatic and custom instrumentation package versions (for example: MSI and NuGet) in sync, and ensure you don't mix major versions of packages.
 </div>
 
 To use custom instrumentation in your .NET application:
-
-1. Add the `Datadog.Trace` [NuGet package][1] to your application.
-2. In your application code, access the global tracer through the `Datadog.Trace.Tracer.Instance` property to create new spans.
+1. Instrument your application using auomatic instrumentation.
+2. Add the `Datadog.Trace` [NuGet package][1] to your application.
+3. In your application code, access the global tracer through the `Datadog.Trace.Tracer.Instance` property to create new spans.
 
 
 [1]: https://www.nuget.org/packages/Datadog.Trace
```

Comment:
Why doesn't it make sense? 🤔 I don't think anything has really changed about that has it?

Reply:
Hmm actually I think it has changed slightly in that the NuGet package _now_ requires automatic instrumentation. But this is pretty similar to my previous [comment](https://github.com/DataDog/documentation/pull/22944/files#r1586594240) that the tabs don't make sense to me anymore because they all have the same two steps of installing automatic instrumentation then add NuGet package so we can probably remove this "NuGet" tab when we remove all the tabs. If this all makes sense to you and you're on board, feel free to resolve this comment thread.